styleguide/_themes/derek/scss/globalElements/mainnav.scss

```diff
@@ -447,4 +469,18 @@
       background: transparent;
     }
   }
+
+  .navbar-nav {
+
+    .navbar-topItem {
+      font-size: 1.45rem;
+
+      .navbar-dropDownLink,
+      .navbar-dropDownLabel,
+      .navbar-tertiary-dropDownLink {
+        border-bottom: 1px solid $gray-darker;
+        font-size: 1.45rem;
+      }
+    }
+  }
```

Comment:
@WidgetsBurritos  Yup this grouping/nesting is already present to fight bootstrap specificity in the live nav sass file
<img width="537" alt="screen shot 2016-09-06 at 3 17 27 pm" src="https://cloud.githubusercontent.com/assets/4554644/18289482/2edf9e0c-7445-11e6-8a00-2595bdc39250.png">